index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+   
+    <link rel="stylesheet" href="CSS/style.css">
+    <link rel="stylesheet" href="/JavaScript/script.js">
+</head>
+<body>
+
+    <!-- Navigation -->
+
+    <header id="header">
+        <img src="Images/logo.jpg" id="header-img">
+
+        <nav id="nav-bar">
+            <ul>
+                <li><a class="nav-link" href="">Features</a></li>
+                <li><a class="nav-link" href="">About</a></li>
+                <li><a class="nav-link" href="">Contact</a></li>
+            </ul>
+
+
+
+           <!-- <a class="nav-link" href="">Features</a>
+           <a class="nav-link" href="">About</a>
+           <a class="nav-link" href="">Contact</a> -->
+
+        </nav>
+      </header> 
+
+      <!-- Form -->
+
+      <form id="form" action="" style="text-align: center;">
+          <h2>Safar And Masud</h2>
+          <input type="email" placeholder="Type your email"  required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+          name="" id="email"  >
+         <input type="Submit" name="" value="submit" id="">
+          
+
+      </form>
+
+      <!-- Video section -->
+
+      <div id="video">
+        <iframe width="800px" height="450" 
+        src="https://www.youtube.com/embed/tgbNymZ7vqY?controls=0">
+        </iframe>
+      </div>
+
+      <!-- Card -->
+      <div class="card" style="width: 100%; height: 600px;">
+        <div class="container" style="width: 30%; height: 400px;">
```

Comment:
It's a good practice to not use inline CSS. Please put all codes related to design in the external CSS file.